multihost_test/bz_automation/script/pamtest.c

```diff
@@ -0,0 +1,140 @@
+#include <security/pam_appl.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <pwd.h>
+
+#define TEST_USER "testUser"
+#define TEST_PASSWORD "TestPassword"
+
+/* Define our service (could really be anything) and then our conversation
+   function. */
+const char *service="pamtest"; 
+extern int converse(int, const struct pam_message **, struct pam_response **, 
+        void *);
+
+int main(int argc, char *argv[])
+{
+        int rval;
+        struct pam_conv pconv;
+        pam_handle_t *phandle;
+        const void *name;
+
+        pconv.conv=&converse;
+        pconv.appdata_ptr=NULL;
+
+/* Call pam_start since you need to before making any other PAM calls */
```

Comment:
Can you indent the comments so that they are aligned with the code?

Reply:
Fixed